File: src/test/java/br/com/model/entities/classes/NovoProdutoFornecedorTest.java
package br.com.model.entities.classes;

import org.junit.jupiter.api.Test;

import br.com.newObjects.model.entities.classes.NovoProdutoFornecedor;

public class NovoProdutoFornecedorTest {
    @Test
    void testNovaListaProdutosFornecedor() {
        System.out.println(NovoProdutoFornecedor.novaListaProdutosFornecedor().toString());
    }
}

File: src/test/java/br/com/model/entities/classes/NovoProdutoFornecedorTest.kt
package br.com.model.entities.classes

import br.com.newObjects.model.entities.classes.NovoProdutoFornecedor
import org.junit.jupiter.api.Test

class NovoProdutoFornecedorTest {
    @Test
    fun testNovaListaProdutosFornecedor() {
        println(NovoProdutoFornecedor.novaListaProdutosFornecedor().toString())
    }
}
